tools/cli/src/commands/wasmatic.rs

```diff
@@ -170,6 +170,93 @@ pub async fn remove(ctx: &AppContext, app_name: String) -> Result<()> {
     Ok(())
 }
 
+#[derive(Deserialize, Debug, Serialize)]
+pub struct InfoResponse {
+    operators: Vec<String>,
+}
+
+pub async fn info(ctx: &AppContext) -> Result<()> {
+    let endpoints = &ctx.chain_info()?.wasmatic.endpoints;
+    let client = Client::new();
+
+    // Make asynchronous requests to all endpoints
+    futures::future::join_all(endpoints.iter().map(|endpoint| {
+        let client = client.clone();
+        async move {
+            // Send the GET request to `/info` endpoint
+            let response = client.get(format!("{}/info", endpoint)).send().await?;
+
+            // Check if the request was successful
+            if !response.status().is_success() {
+                bail!("Error: {:?}", response.text().await?);
+            }
+
+            let info_response: InfoResponse = response.json().await?;
+
+            println!(
+                "Output for operator `{endpoint}`: {}",
+                serde_json::to_string_pretty(&info_response)?
+            );
+
+            Ok(())
+        }
+    }))
+    .await
+    .into_iter()
+    .collect::<Result<(), _>>()
+}
+
+// Define the structure to deserialize the response
+#[derive(Deserialize, Debug, Serialize)]
+pub struct AppResponse {
+    apps: Vec<AppInfo>,
+    digests: Vec<String>,
+}
+
+#[derive(Deserialize, Debug, Serialize)]
+pub struct AppInfo {
+    name: String,
+    digest: String,
+    trigger: Trigger,
+    permissions: Value,
+    testable: bool,
+}
+
+#[derive(Deserialize, Debug, Serialize)]
+pub struct Queue {
+    task_queue_addr: String,
+    hd_index: u32,
+    poll_interval: u32,
+}
+
+pub async fn app(ctx: &AppContext) -> Result<()> {
+    let endpoints = &ctx.chain_info()?.wasmatic.endpoints;
+    let client = Client::new();
+
+    // Call the first operator in the list (change this if you want to call a specific one)
+    let endpoint = &endpoints[0]; // You can also specify a different index
```

Comment:
Not a blocker, can be a followup PR - but can we make this an optional argument in the commandline?

Reply:
@ueco-jb from my perspective, up to you if you want to do that and then merge or just merge what's here now and come back to that later

Reply:
A good idea, I added that.